### DISCUSSION TITLE
Push to frontend

### DISCUSSION THREAD

**Discussion by wlwatkins** [wailsapp/wails#1840] [Q&A]
How would you push data to front end?
For example if I have a menu tab that calls an action in go, how would I relay the event to the front end such as changing the CSS of an object. 
The only solution I currently have is to probe the backend from the front end every n second but this sounds quite heavy.

**Comment by leaanthony** [comment]
Events would be the best way at the moment: https://wails.io/docs/reference/runtime/events
